src/components/TodoList.js

```diff
@@ -1,36 +1,53 @@
 import React, { useEffect } from 'react'
 import ListItem from './ListItem'
-import { connect } from 'react-redux'
-import { getTodos } from '../store/todos'
+
+/*
+  TodoList component
+  - Function: TodoList is a presentational component that displays a list of todos as
+              the user adds them
+    Props:    It receives the following props from the Homepage component:
+              - todos - list of todos held in state (Redux store)
+              - editTodo - edits a single todo list item
+              - deleteTodo - deletes a todo from the list
+              - refreshTodos - updates to the current list of todos after a change been made
+              - getTodo - fetches a single todo item
+    Children: ListItem
+                - ListItem will receive as props from TodoList the following:
+                    - all of the properties of each individual todo
+                          - id,
+                          - task,
+                          - completed
+                    - editTodo - edits a single todo item
+                    - deleteTodo - deletes a single todo item
+                    - getTodo - fetches a single todo from the Redux store. This item will also
+                                correspond to the react router route ('/todos/:id) with id being
+                                the id of the todo item
+
+*/
 
 const TodoList = (props) => {
+  const { todos, editTodo, deleteTodo, getTodo, refreshTodos } = props
   useEffect(() => {
-    async function getData () {
-      await props.getTodos()
+    async function fetchTodos () {
+      await refreshTodos()
     }
-    getData()
-  })
-
-  const { todos } = props
```

Comment:
This seems more appropriate for the `Homepage` component to do. Then`Homepage` passes `todos` down to `TodoList`. `TodoList` should only ever need to call `refreshTodos` if the user hard refreshes ie. presses a "refresh" button or something. Unless I am missing something.

Reply:
I was thinking about adding a refresh button or something similar

Reply:
Refresh button sounds good, but the hook that initializes the todos should probably be moved to the `Homepage`, correct?